### ISSUE TITLE
PUT - "store under" vs "store at" (Trac #112)

### ISSUE THREAD

**Issue by mnot** [httpwg/httpbis-issues#112]
```text
8.6.  PUT
    The PUT method requests that the enclosed entity be stored under the
    supplied Request-URI.
```

Several people thought "under" implies "at a subpath of the Request-URI". To reduce confusion, this should be changed from "under" to "at".

Migrated from https://trac.ietf.org/ticket/112
```json
{
    "status": "closed",
    "changetime": "2011-03-10T08:40:47",
    "_ts": "1299746447000000",
    "description": "{{{\n8.6.  PUT\n    The PUT method requests that the enclosed entity be stored under the\n    supplied Request-URI.\n}}}\n\nSeveral people thought \"under\" implies \"at a subpath of the Request-URI\". To reduce confusion, this should be changed from \"under\" to \"at\".",
    "reporter": "julian.reschke@gmx.de",
    "cc": "",
    "resolution": "fixed",
    "time": "2008-04-10T16:39:15",
    "component": "p2-semantics",
    "summary": "PUT - \"store under\" vs \"store at\"",
    "priority": "",
    "keywords": "",
    "milestone": "03",
    "owner": "julian.reschke@gmx.de",
    "type": "editorial",
    "severity": "Active WG Document"
}
```

**Comment by mnot** [comment]
Trac update at 20080410T16:39:27: julian.reschke@gmx.de changed owner from "" to "julian.reschke@gmx.de"

**Comment by mnot** [comment]
Trac update at 20080410T19:12:20: 
* julian.reschke@gmx.de commented:

> Fixed in [241]:
> 
> Resolve #112: replace "to be stored under the supplied Request-URI" by "... at ....". (closes #112).


* julian.reschke@gmx.de changed resolution from "" to "fixed"
* julian.reschke@gmx.de changed status from "new" to "closed"

**Comment by mnot** [comment]
Trac update at 20110310T08:40:47: fielding@gbiv.com commented:

> From [1158]:
> 
> Replaced the general prohibition on unrecognized Content-* header fields
> with a specific prohibition of Content-Range (the only field for which
> it is an actual problem) and a general requirement regarding checking
> for consistency.  Unfortunately, this required rewriting the entire
> section on PUT to get rid of the misconceptions about storing resources
> and reflect how PUT is actually implemented in practice.
> 
> Addresses #79, #102, #103, #104, #112, #180, #231, and #267
>
